pkg/analytics/track.go

```diff
@@ -69,6 +69,7 @@ const (
 	disableEvent             = "Disable Analytics"
 	stackNotSupportedField   = "Stack Field Not Supported"
 	buildPullErrorEvent      = "BuildPullError"
+	deleteMultipleContexts   = "Delete Multiple Contexts"
```

Comment:
```suggestion
	deleteContexts   = "contexts deletion"
```
as event name I would use something more descriptive in term of event identifier and use a variable which "allow" only the deletion of one context

Reply:
> nit: as event name I would use something more descriptive in term of event identifier and use a variable which "allow" only the deletion of one context